parambokeh/util.py

```diff
@@ -118,7 +120,15 @@ def process_plot(plot, doc, plot_id, comm):
         from holoviews import renderer
         renderer = renderer('bokeh').instance(mode='server' if comm is None else 'default')
         plot = renderer.get_plot(plot, doc=doc)
-
+    elif plot.__class__.__name__ == 'Figure' and hasattr(plot, '_cachedRenderer'):
+        bytes_io = BytesIO()
+        plot.canvas.print_figure(bytes_io)
+        data = bytes_io.getvalue()
+        b64 = base64.b64encode(data).decode("utf-8")
+        src = "data:image/png;base64,{b64}".format(b64=b64)
+        html = "<img src='{src}'></img>".format(src=src)
+        width, height = plot.canvas.get_width_height()
+        return Div(text=html, width=width, height=height)
     if not hasattr(plot, '_update_callbacks'):
         raise ValueError('Can only render bokeh models or HoloViews objects.')
```

Comment:
Presumably this error message is out of date.  Maybe "Not one of the currently supported displayable objects (Bokeh models, HoloViews objects, or Matplotlib figures)."?

Would it be possible to have a fallback that allows any object with a `_repr_html_` method?  That would cover Pandas `.head()` and possibly streamz objects, and would let users make their own wrappers around other things...

Reply:
>Would it be possible to have a fallback that allows any object with a _repr_html_ method?

Sure.

>and possibly streamz objects

Definitely not, those work using ipywidgets.